packages/monitoring-system/data-processor/test/unit/data-processors/cloud-logs-data-processor.spec.ts

```diff
@@ -248,64 +256,209 @@ describe('Cloud Logs Processor', () => {
       });
 
       describe('trigger information logs', () => {
-        const executionRecord = {
-          document: {
-            '1lth8bxqr88v': {
-              execution_id: '1lth8bxqr88v',
+        describe('GitHub trigger', () => {
```

Comment:
Instead of generic tests for all trigger types, I added separate tests for `GitHub Webhook`, `Cloud Tasks`, `Cloud Scheduler` and `Pub/Sub` triggers